src/App.js

```diff
@@ -3,20 +3,44 @@ import { useEffect } from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
 
+
 function App() {
   const [todoList, setTodoList] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
+  
+  
+  async function fetchData() {
+    const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
 
-  useEffect(() => {
-    new Promise((resolve, reject) => {
-      setTimeout(() => {
-        resolve({ data: { todoList: JSON.parse(localStorage.getItem('savedTodoList')) || [] } }); 
-      }, 2000); 
-    })
-    .then((result) => {
-      setTodoList(result.data.todoList);
+    const options = {
+      method: 'GET',
+      headers: {
+        'Authorization': `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`
+      }
+    }
+
+    try {
+      const response = await fetch(url, options);
+      if (!response.ok) {
+        throw new Error(`Error: ${response.status}`);
+      };
+
+      const data = await response.json();
+
+      const todos = data.records.map(record => ({
+        title: record.fields.Title, 
+        id: record.id
+      })); 
```

Comment:
**Title** property doesn't exist in the record.fields object. That typo is causing rendering todos with undefined title properties. Fix it.